CMakeLists.txt

```diff
@@ -222,32 +229,69 @@ if(NOT NETWORKIT_EXT_TLX)
 	endif()
 else()
 	add_library(tlx STATIC IMPORTED)
-	set_target_properties(tlx PROPERTIES
-			IMPORTED_LOCATION "${NETWORKIT_EXT_TLX}/lib/libtlx.a"
-			INTERFACE_INCLUDE_DIRECTORIES "${NETWORKIT_EXT_TLX}/include/")
+	if(WIN32)
+		set_target_properties(tlx PROPERTIES
+				IMPORTED_LOCATION "${NETWORKIT_EXT_TLX}/lib/tlx.lib"
+				INTERFACE_INCLUDE_DIRECTORIES "${NETWORKIT_EXT_TLX}/include/")
+	else()
+		set_target_properties(tlx PROPERTIES
+				IMPORTED_LOCATION "${NETWORKIT_EXT_TLX}/lib/libtlx.a"
+				INTERFACE_INCLUDE_DIRECTORIES "${NETWORKIT_EXT_TLX}/include/")
+	endif()
```

Comment:
Instead of duplicating these lines, maybe it's better to define a `${NETWORKIT_LIB_SUFFX}`? Maybe CMake even defines such a variable already.

Reply:
Good point. Had something like that in a previous version (introducing new if/else-cases), but this did not include a cmake variable.